packages/customWidgets/calendar-web/src/components/CalendarContainer.ts

```diff
@@ -107,22 +108,29 @@ export default class CalendarContainer extends Component<Container.CalendarConta
         return !this.props.mxObject || !this.props.editable || this.props.readOnly;
     }
 
-    private getStartPosition = async (mxObject: mendix.lib.MxObject): Promise<Date> => {
-        if (this.props.startDateAttribute && mxObject) {
-            return new Promise(resolve => {
-                const processStartDateAttributeValue = (startDateAttributeValue: number | ""): void => {
-                    if (startDateAttributeValue) {
-                        resolve(new Date(startDateAttributeValue));
-                    }
+    private extractAttributeValue = async <WidgetPropertyTypes>(
+        mxObject: mendix.lib.MxObject,
+        attributePath: string
+    ): Promise<WidgetPropertyTypes | "" | null | undefined> => {
+        if (!attributePath) {
+            return Promise.resolve(undefined);
+        }
 
-                    resolve(new Date());
-                };
+        return new Promise(resolve => {
+            mxObject.fetch(attributePath, (attributeValue: any): void => resolve(attributeValue));
+        });
+    };
 
-                mxObject.fetch(this.props.startDateAttribute, processStartDateAttributeValue);
-            });
+    private getStartPosition = async (mxObject: mendix.lib.MxObject): Promise<Date> => {
+        if (mxObject) {
+            const startDateAttributeValue = await this.extractAttributeValue<number>(
```

Comment:
Similar comment to Diego's, any error will not be handled

Reply:
Fixed